modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -10,7 +10,7 @@ A key feature to enable as part of a {sno} installation is workload partitioning
 
 [NOTE]
 ====
-Workload partitioning must be applied during installation.
+You can only enable workload partitioning during the cluster installation process. You cannot disable workload partitioning post-installation. However, you can reconfigure workload partitioning by updating the `cpu` value that you define in the `performanceprofile`, and in the MachineConfig CR in the following procedure. 
```

Comment:
s/only enable workload partitioning /enable workload partitioning only   --or--  enable workload partitioning during the cluster installation process only